packages/adapter-netlify/README.md

```diff
@@ -1,6 +1,8 @@
 # adapter-netlify
 
-Adapter for Svelte apps that creates a Netlify app, using serverless functions for dynamically generating pages.
+A SvelteKit adapter that creates a Netlify app.
+
+If you're using [adapter-auto](../adapter-auto), you don't need to install this unless you need to specify Netlify-specific options, since it's already included.
```

Comment:
i'm not an English major, but do we need another comma here? the reason I think that is that you could take out the "unless..." section and it would read "you don't need to install this since it's already included". (if we do change it then it'd need to be changed in the others as well)
```suggestion
If you're using [adapter-auto](../adapter-auto), you don't need to install this, unless you need to specify Netlify-specific options, since it's already included.
```

Reply:
I think it's fine as-is. You'd never have three commas in a sentence as short as this one unless you're Russ Hanneman
 
![image](https://user-images.githubusercontent.com/1162160/151863175-6b6edea9-7c14-418c-ac2b-9e26cb1e413b.png)